src/pages/board/[slug]/customize.tsx

```diff
@@ -58,6 +59,9 @@ export default function CustomizationPage() {
   const { t } = useTranslation('boards/customize');
   const form = useBoardCustomizationForm({
     initialValues: {
+      access: {
+        allowAnonymous: config?.settings.access.allowAnonymous ?? false
```

Comment:
Why is it config?. ? Shouldn't the config be loaded server-side and served as props instead ?

Reply:
We use tRPC to refresh during the Client. Updates on the server would otherwise not get refreshed (unless you re-render on the server).

Reply:
> Why is it config?. ? Shouldn't the config be loaded server-side and served as props instead ?

Yeah, that is what it's doing?